openupgradelib/openupgrade_merge_records.py

```diff
@@ -217,10 +217,11 @@ def _change_reference_refs_sql(
         try:
             model = env[row[0]]
         except KeyError:
-            continue
-        if not model._auto:  # Discard SQL views
-            continue
-        table = model._table
+            table = get_model2table(row[0])
+        else:
+            if not model._auto:  # Discard SQL views
```

Comment:
This is not evaluated in the proper place.

Reply:
What do you mean? The `else` part is executed if no exception is triggered. Thus, the code is fine.